src/bitcoin.udl

```diff
@@ -30,6 +30,11 @@ enum Network {
   "Regtest",
 };
 
+dictionary TxOut {
+  u64 value;
```

Comment:
Because the `rust-bitcoin` `TxOut` uses `Amount` I think we should also use it here.

Reply:
indeed, thanks! will change rn